README.md

```diff
@@ -79,11 +80,12 @@ period = 60
 ```
 
 #### [Astarte Message Hub](https://github.com/astarte-platform/astarte-message-hub)
+
 A central service that runs on (Linux) devices for collecting and delivering messages from N apps
 using 1 MQTT connection to Astarte.
 
-**N.B.** When using this option the Astarte Message Hub should already be installed and running
-on your system.
+**N.B.** When using this option the Astarte Message Hub should already be installed and running on
```

Comment:
I would add a comma after _option,_ to improve readability.

Reply:
Done